Gems/ROS2/Code/Source/VehicleDynamics/Utilities.cpp

```diff
@@ -191,4 +191,24 @@ namespace ROS2::VehicleDynamics::Utilities
         return AZ::EntityComponentIdPair(wheelEntityId, hingeComponent->GetId());
     }
 
+    float RampVelocity(float targetVelocty, float lastVelocity, AZ::u64 deltaTimeNs, float acceleration, float maxVelocity)
+    {
+        const float eps = 0.01f*acceleration;
```

Comment:
nit: Please update variable name (or add a comment) to make `eps` clearer

Reply:
uber nit: Also it's generally preferred (mentioned in the coding standards) to include white space around operators (clang-format should fix this I think)

Reply:
Applied clang, added comment.